aiohttp/helpers.py

```diff
@@ -423,6 +423,9 @@ def content_disposition_header(
 def is_expected_content_type(
     response_content_type: str, expected_content_type: str
 ) -> bool:
+    """Checks if received content type is processable as an expected one.
+    Both arguments should be given without parameters.
```

Comment:
Per PEP 257, there must always be an empty line between the docstring title and the description part.
```suggestion

    Both arguments should be given without parameters.
```

Reply:
BTW if PEP 257 is desired here, I suggest adding flake8-docstrings to pre-commit. I can submit a PR for that, let me know.

Reply:
Maybe it is a good idea.
To the node: I'm not volunteered to fix all broken docstrings if the check is applied :)